test/inventoryDb/fixtures.js

```diff
@@ -0,0 +1,55 @@
+const AVAILABILITY_YEAR_MONTH = '2018-03';
+const SINGLE_AVAILABILITY_DAY = '26';
+const NO_AVAILABILITY_DAY = '12';
+const AVAILABLE_RANGE_START_DAY = '19';
+const AVAILABLE_RANGE_END_DAY = '22';
+
+const availableDateRange = [];
+for (let d = parseInt(AVAILABLE_RANGE_START_DAY, 10);
+  d <= parseInt(AVAILABLE_RANGE_END_DAY, 10); d += 1) {
```

Comment:
I don't understand this for loop?

Reply:
Airbnb linting complained about the length of this line, hence the broken up for loop.